files/fr/glossary/continuous_media/index.html

```diff
@@ -0,0 +1,9 @@
+---
+title: Médias Continus
```

Comment:
In French don't capitalize each word of a title like in English. Capitalize only the first word (and proper nouns) :)